docs/markdown/snippets/fallback_bool.md

```diff
@@ -0,0 +1,5 @@
+## `dependency` accepts a boolean argument
+
+Passing `fallback: true` to `dependency` enables automatic fallback to
+a subproject with the same name as the dependency, while `fallback: false`
+blocks automatic search for a subproject that can fulfill the dependency.
```

Comment:
It's not clear what 'automatic' means here (to people not aware of this feature) and it's not only about 'same name' but more generally a subproject that _provides_ a dependency.

I would simply copy/paste the text suggested in the doc above:
```
`fallback` keyword argument now accept boolean value. If `true` and the dependency is not found on the system, Meson will fallback to a subproject that provides this dependency, even if the dependency is optional. If `false` Meson will not fallback even if a subproject provides this dependency. See [Wrap documentation](Wrap-dependency-system-manual.md#provide-section).
```